tests/benchmarks/experiments/test_e2e_mini.py

```
@@ -0,0 +1,189 @@
+    results = []
+    
+    for i, qa in enumerate(qa_pairs):
+        question = qa['question']
+        expected = str(qa.get('answer', ''))  # Convert to string
+        category = qa.get('category', 1)
+        
+        print(f"\n  Q{i+1}: {question[:50]}...")
+        print(f"      Expected: {expected[:40]}...")
+        
+        # Recall memories
+        resp = requests.get(
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)